portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,34 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+async function addRandomGreeting() {
+  const responseFromServer = await fetch('/hello');
+  const textFromResponse = await responseFromServer.json();
+  const greetingContainer = document.getElementById('greeting-container');
+  //const listText = createListElement(textFromResponse);
```

Comment:
Can you delete? Generally, submitting code you're no longer using just clutters your file. It should be in your version history anyway if you need to go back and find it :)